trustymail/domain.py

```diff
@@ -90,6 +116,10 @@ def generate_results(self):
 
                         "MX Record": self.has_mail(),
                         "Mail Servers": self.format_list(self.mail_servers),
+                        "Mail Server Ports Tested": self.format_list(list(map(lambda x:str(x), self.ports_tested))),
+                        "Mail Server Is Listening": self.format_list(list(map(lambda x:x + ";" + str(self.starttls_results[x]["is_listening"]), self.starttls_results.keys()))),
```

Comment:
If we're really interested in SMTP support, what advantage does knowing `Mail Server Is Listening` bring? Does this value need to be exposed in the output?

Reply:
It doesn't have to be exposed.  I just thought it may be useful to know whether something is actually listening in some edge cases.  I'm fine removing it if you prefer that.

Reply:
@h-m-f-t, did you decide that this change is something you want?  If so I can make the change.